e2e/cypress/tests/integration/enterprise/ldap_group/channel_modes_spec.ts

```diff
@@ -18,7 +19,10 @@ describe('LDAP Group Sync - Test channel public/private toggle', () => {
         cy.apiRequireLicenseForFeature('LDAPGroups');
 
         // Enable LDAP and LDAP group sync
-        cy.apiUpdateConfig({LdapSettings: {Enable: true}});
+        cy.apiGetConfig().then(({config}) => {
+            set(config, 'LdapSettings.Enable', true);
+            cy.apiUpdateConfig(config);
+        });
```

Comment:
Sorry. I just submitted a PR to revert the use of `lodash.set` in favor of [`DeepPartial` as in `apiUpdateConfig(newConfig: DeepPartial<AdminConfig>)`](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/api/system.d.ts#L100). See https://github.com/mattermost/mattermost-webapp/pull/11362. Use of DeepPartial is already on master. Please update accordingly. Thanks!

Reply:
Sure. Will do that